modules/publisher/views.py

```diff
@@ -116,22 +116,12 @@ def post_agreement():
     if agreed == 'on':
         try:
             api.post_agreement(flask.session, True)
-        except ApiTimeoutError as api_timeout_error:
-            flask.abort(504, str(api_timeout_error))
-        except ApiConnectionError as api_connection_error:
-            flask.abort(502, str(api_connection_error))
-        except ApiResponseDecodeError as api_response_decode_error:
-            flask.abort(502, str(api_response_decode_error))
         except ApiResponseErrorList as api_response_error_list:
             codes = [error['code'] for error in api_response_error_list.errors]
```

Comment:
Why is not `_handle_errors_list` used here as well?

Reply:
`_handle_errors_list` handles known errors and returns an `abort` while for those functions I don't too.